qa/python_models/dlpack_test/model.py

```diff
@@ -37,12 +37,11 @@ def test_pytorch_dlpack(self):
         # Test different dtypes
         pytorch_dtypes = [
             torch.float16, torch.float32, torch.float64, torch.int8,
-            torch.int16, torch.int32, torch.int64, torch.uint8, torch.bool
```

Comment:
Looks like after the PyTorch version upgrade it no longer supports torch.bool for dlpack: https://github.com/pytorch/pytorch/commit/1cb3507ed38330af43bf23e255dc64e9215384c4

CC @GuanLuo